app/views/certificate/VatCertificate.scala.html

```diff
@@ -81,6 +81,16 @@
   List("1", "Z1").contains(model.businessTypeMsgKey.split('.').last)
 }
 
+    @webchatLink = {
+        <br>
+        <br>
```

Comment:
Can you use the GDS classes to add padding please? <br> can make it difficult for screen readers
https://design-system.service.gov.uk/styles/spacing/